src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -424,6 +424,7 @@
   </ItemGroup>
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS'">
     <Reference Include="Microsoft.Win32.Primitives" />
+    <Reference Include="System.Buffers" />
```

Comment:
Is this needed? This probably came in because of ArrayPool usage and I don't see the code using this dependency.

Reply:
I just removed..